docs/source/index.rst

```diff
@@ -107,6 +107,13 @@ contributing to the project, helping other users and discuss all things MLOps.
    GitHub <https://github.com/bentoml/Yatai>
    Blog <https://modelserving.com>
 
+.. toctree::
+   :caption: Ecosystem
+   :hidden:
+
+   🍱 BentoML <https://github.com/bentoml/BentoML>
+   ☁️ BentoML Cloud <https://www.bentoml.com/bento-cloud/>
```

Comment:
```suggestion
   ☁️ BentoCloud <https://www.bentoml.com/bento-cloud/>
```

Reply:
@jianshen92 let's make sure the naming is consistent for BentoCloud, let's also change it in the BentoML docs sidebar next time we update the bentoml docs